ggml-llama-via-wasinn/ggml-llama-wasm/Cargo.toml

```diff
@@ -4,4 +4,4 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-wasi-nn = { git = "https://github.com/second-state/wasmedge-wasi-nn", branch = "ggml" }
+wasmedge-wasi-nn = { git = "https://github.com/second-state/wasmedge-wasi-nn", branch = "ggml" }
```

Comment:
It would be better to revise it to `wasi-nn = { package = "wasmedge-wasi-nn", git = "https://github.com/second-state/wasmedge-wasi-nn", branch = "ggml" }`

Reply:
Sure !  , I have done the changes as you have suggested